tests/testdata/default_config.yaml

```diff
@@ -47,9 +47,9 @@ generate:
       max_ctx_size: 4096
     model_path: /cache/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf
     vllm:
+      gpus: null
```

Comment:
(Observation) I wonder why these were not sorted before. We dump with `sort_keys=True`... The file was probably manually updated before.

Reply:
I think this was changed by my linter and I didn't catch it before it got committed.